images/openstack-cli/Earthfile

```diff
@@ -0,0 +1,29 @@
+VERSION 0.8
+
+platform-image:
+  ARG RELEASE=2023.1
+  FROM ../cloud-archive-base+image --RELEASE=${RELEASE}
+  DO ../+APT_INSTALL --PACKAGES "\
+    python3-barbicanclient \
+    python3-designateclient \
+    python3-glanceclient \
+    python3-heatclient \
+    python3-magnumclient \
+    python3-manilaclient \
+    python3-neutronclient \
+    python3-novaclient \
+    python3-octaviaclient \
+    python3-openstackclient \
+    python3-osc-placement \
+    python3-senlinclient \
+    python3-swiftclient"
+#  ARG REGISTRY=ghcr.io/vexxhost/atmosphere
+  ARG REGISTRY=10.13.55.223:5000
```

Comment:
FYI, in your testing, you can keep this as is but run `earthly ./images/openstack-cli+platform-image --REGISTRY=10.13.55.223:5000`

Reply:
(Follow-up thought, what do you think of installing the CLI from source instead of packages?)

Reply:
I like this idea better. Will do.